xbmc/GUIInfoManager.cpp

```diff
@@ -5382,6 +5382,13 @@ bool CGUIInfoManager::GetItemBool(const CGUIListItem *item, int condition) const
       }
       else if (pItem->HasEPGInfoTag())
       {
+        CEpgInfoTagPtr epgTag = pItem->GetEPGInfoTag();
+
+        // Check if the tag has a currently active recording associated
+        if (epgTag->HasRecording() && epgTag->IsActive())
+          return true;
+
+        // Search all timers for something that matches the tag
         CFileItemPtr timer = g_PVRTimers->GetTimerForEpgTag(pItem);
         if (timer && timer->HasPVRTimerInfoTag())
```

Comment:
Good idea, I haven't checked. I noticed this piece of code is basically run constantly (but a breakpoint here and it will be hit immediately while in the guide window), either something is very wrong or we should really attempt to do as little as possible here.